### ISSUE TITLE
Rename to "VR Grip"

### ISSUE THREAD

**Issue by JScott** [JScott/ViveGrip#20]
I've tested this with Oculus Rift+Touch and it works. I should probably change the name to make that obvious. Marketing is important! Or so I hear.

**Comment by JScott** [comment]
It's probably worth waiting until I have native Oculus SDK support. Some platforms require the Oculus SDK for submission and I'm not sure that OpenVR makes the cut.

**Comment by oscarqht** [comment]
Does this also work for Windows MR?

**Comment by JScott** [comment]
Good question! I don't have a Windows MR device to test it but Vive Grip is built on SteamVR and [I thiiiink Windows MR supports that](https://docs.microsoft.com/en-us/windows/mixed-reality/enthusiast-guide/using-steamvr-with-windows-mixed-reality)? The Rift+Touch happened to work out of the box so I wouldn't be surprised.

**Comment by oscarqht** [comment]
Yes. SteamVR unity plugin works for Windows MR too. But not out of the box, you need to install this [Windows Mixed Reality for SteamVR](http://store.steampowered.com/app/719950/Windows_Mixed_Reality_for_SteamVR/) though. I'll try ViveGrip on WindowsMR tonight, see if it works :)

**Comment by JScott** [comment]
Oh wow, thanks! I appreciate the check whenever you have a chance to get around to it.

To get a bit more detailed, as long as it responds to `SteamVR_Controller.Device`'s `GetPressDown`, `GetPressUp`, and `GetPress` for [these inputs](https://github.com/JScott/ViveGrip/blob/master/Assets/ViveGrip/Scripts/Prefab/ViveGrip_ControllerHandler.cs#L113-L118) then it should be fine. I don't know how the WinMR controller is laid out though so maybe there'll be some unexpected results.

**Comment by oscarqht** [comment]
FYI

![Windows MR Controller](https://user-images.githubusercontent.com/165961/37387460-6c4a02f8-2797-11e8-9730-1ac2012e98e7.png)

1. The buttons, layout and function of the buttons are basically the same as VIVE controller.
2. With one exception that there's a thumbstick besides the touchpad. Usually the thumbstick is used to teleport around while the touchpad is for scrolling UI elements. The thumbstick also has a click event.
3. Also the WMR controller is lighter and smaller than the VIVE's.

**Comment by oscarqht** [comment]
Just tested ViveGrip with Windows Mixed Reality (Samsung Odyssey HMD), it works! Objects in the example scene could be picked up by holding the grip button and used by clicking the trigger button. You're now officially VRGrip :D

**Comment by JScott** [comment]
Fantastic! Thank you so much for letting me know :D

I feel like it's more SteamVRGrip at this point because it doesn't have native WinMR or Rift support which causes significant barriers, like getting your app published on Oculus Home. Still, it's very exciting to hear that it functions across the board, opening up the opportunities for creators.

I guess at the very least I should update the readme to list WinMR.
